servers/visual/visual_server_viewport.cpp

```diff
@@ -260,15 +265,25 @@ void VisualServerViewport::_draw_viewport(Viewport *p_viewport) {
 }
 
 void VisualServerViewport::draw_viewports() {
-
-	//sort viewports
-
-	//draw viewports
+	// get our arvr interface in case we need it
+	Ref<ArVrInterface> arvr_interface = ArVrServer::get_singleton()->get_primary_interface();
+	if (arvr_interface.is_valid()) {
+		// update our positioning information as late as possible...
+		float delta = 0; ///@TODO need to set this, time accurately since last time we called this!
```

Comment:
@reduz is there anything I can call here to determine how much time has passed since the last time this was called? I noticed there where some timing functions in OS but I couldn't tell if they were implemented for all platforms.